### ISSUE TITLE
Build configuration "Debug" not work

### ISSUE THREAD

**Issue by xunil82** [LocalJoost/BlogComments#374]
Hi joost,
great work and perfect described.
The source works with Master configuration, but not with Debug. I would change something and debug the result. Unfortunately the App only starts but does not recognize qr Codes. I see only the debug logger and the visu. My build configuration like your named in the last post. Hope you can help me.

**Comment by LocalJoost** [comment]
This is correct. Only the Master config works. I have no idea why - I presume because the Microsoft packages don't support that. They are still in preview. You might try to upgrade via the package manager. If that doesn't help I suggest you file a bug with Microsoft 😊

**Comment by xunil82** [comment]
Thanks for this information and your great blog posts.
